*** Starting uWSGI 2.0.19.1 (64bit) on [Thu Jun 24 11:39:03 2021] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.21) on 24 June 2021 02:18:18
os: Darwin-20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64
nodename: llshdeMacBook-Pro.local
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /Users/llsh/PycharmProjects/pythonProject/dataAnalysis/dataAnalysis
writing pidfile to uwsgi.pid
detected binary path: /usr/local/bin/uwsgi
chdir() to /Users/llsh/PycharmProjects/pythonProject/dataAnalysis
your processes number limit is 2784
your memory page size is 4096 bytes
detected max file descriptor number: 256
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:62757 (port auto-assigned) fd 3
Python version: 3.9.5 (default, May  4 2021, 03:36:27)  [Clang 12.0.0 (clang-1200.0.32.29)]
Python main interpreter initialized at 0x7fb39c509970
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416640 bytes (406 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "dataAnalysis/wsgi.py", line 12, in <module>
    from django.core.wsgi import get_wsgi_application
ModuleNotFoundError: No module named 'django'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 63761)
spawned uWSGI worker 1 (pid: 63766, cores: 2)
spawned uWSGI worker 2 (pid: 63767, cores: 2)
spawned uWSGI worker 3 (pid: 63768, cores: 2)
spawned uWSGI worker 4 (pid: 63769, cores: 2)
spawned uWSGI http 1 (pid: 63770)
--- no python application found, check your startup logs for errors ---
[uwsgi-http key: 127.0.0.1:8000 client_addr: 127.0.0.1 client_port: 2810] hr_instance_read(): Connection reset by peer [plugins/http/http.c line 647]
[pid: 63768|app: -1|req: -1/1] 127.0.0.1 () {34 vars in 459 bytes} [Thu Jun 24 12:57:19 2021] POST /api/apriori => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (1 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 63767|app: -1|req: -1/2] 127.0.0.1 () {34 vars in 459 bytes} [Thu Jun 24 12:58:00 2021] POST /api/apriori => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
respawned uWSGI http 1 (pid: 64104)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 64104)
worker 4 buried after 1 seconds
worker 3 buried after 1 seconds
worker 2 buried after 1 seconds
worker 1 buried after 1 seconds
goodbye to uWSGI.
